content/post/release-of-1.4.0.md

```diff
@@ -0,0 +1,189 @@
+---
+date: "2018-01-25T12:00:00+00:00"
+author: "thehowl"
+title: "Gitea 1.4 is released"
+tags: ["release"]
+draft: true
+---
+
+Hello! Yet another release cycle is over; this time around we merged [X pull requests](https://github.com/go-gitea/gitea/pulls?utf8=%E2%9C%93&q=is%3Apr+milestone%3A1.4.0+is%3Amerged) - we made sure to deliver as much as we could in this new Gitea version, and we're sure you're going to like it.
+
+Starting from this release, we'll also make sure to walk you through the most important changes we made - instead of just dumping the PRs closed in this release and their category. And boy have we got a lot of stuff to talk about.
+
+<!--more-->
+
+## Reactions ([#2856](https://github.com/go-gitea/gitea/pull/2856))
+
+![Screenshot of Reactions](/demos/2856/1.png)
+
+![Screenshot of Reactions](/demos/2856/2.png)
+
+Yes. Yes! YES!!! Gitea now has reactions on pull requests and issues. Now hopefully your collaborators will shut up with their "+1" comments on the issue and will instead just press the 👍 button. Oh yeah 😎.
+
+_Thanks to **[@lafriks](https://github.com/lafriks)**_
+
+## Responsive UI ([#2750](https://github.com/go-gitea/gitea/pull/2750))
+
+![Responsive UI screenshot](/demos/2750/1.png)
+
+![Responsive UI screenshot](/demos/2750/2.png)
+
+![Responsive UI screenshot](/demos/2750/3.png)
+
+Starting from version 1.4.0, Gitea's web interface is finally responsive; this means it can be properly used from your phone without having to pinch everywhere to see what is on the screen! There may be some places where the interface is still not completely responsive, in which case we ask you kindly to [file an issue](https://github.com/go-gitea/gitea/issues) if there isn't one already, posting screenshots of what does not work.
```

Comment:
"finally responsive" suggests that we'll keep it a priority to make sure it stays taht way. AFAIK we don't have any tests that checks the frontend right? 🤔

Reply:
No, we don't really have those. (If someone is willing to do them, I'll be glad. I can bear writing unit tests for Go, definitely not writing tests for the frontend). So what do you suggest?

Reply:
You might prefer property testing. 
https://github.com/leanovate/gopter

Reply:
@thehowl I'd suggest something like "Starting from version 1.4.0, we've reworked some of the interface to allow for a more responsive design" maybe? 🤔

Reply:
With some examples?

Reply:
@ShalokShalom examples would be great but not necessarily